### PR TITLE
Business Acumen Interpretation notes

### PR DESCRIPTION
**Pull request by craigatproperly**
Added comments about how awareness of real estate and awareness of tech can be effective at the emerging level, with illustrative (not comprehensive) examples of the types of context that could be valued. 

Also did the same for how depth and nuance in those same areas at the proficient level can help people be more effective, in particular in planning and design both technically and as a partner to product and design (and as being a generator of ideas within Properly)

### CODE REVIEW

**Comment by mruthven** [comment]
Have read through this, like the general approach. I would like to come back to it and read again so that I can maybe add some valuable comments.

File: competencies/roles/_TechWideGeneral.md [hunk 1]
```diff
@@ -122,3 +122,76 @@
         </ul></td>
     </tr>
 </table>
+
+### Interpreting Business Acumen
+
+Business acumen in a technology team context has two main dimension, one is core subject of
```
Comment (mruthven) [inline competencies/roles/_TechWideGeneral.md:128]: nit: `two main dimensions`

File: competencies/roles/_TechWideGeneral.md [hunk 2]
```diff
@@ -122,3 +122,76 @@
...
+##### Real Estate
+Aware of the most relevant aspects of Canadian real estate in order to contextualize the 
+broader work that we are doing. Examples of the type of information: 
+- the agent system and fee structure (percent of cost, billed on sell side, billed from buyer agent to seller agent)
+- the reasons why buying first can be hard 
+  - risk of 2 homes and qualifying for 2 mortgages, unknown value of home you 
+  are selling 
+  - risk of 0 homes and not getting a suitable new home, managing the gap time
+- Understanding our system (estimating the sale price, discounting to unlock equity)
+
```
Comment (mruthven) [inline competencies/roles/_TechWideGeneral.md:142]: I know this list isn't meant to be exhaustive, but I would suggest that an understanding of the fact that we source our listing data from third parties (implying need for interpretation, shaping, cleaning, etc) would be a valuable example of emerging competence here.

File: competencies/roles/_TechWideGeneral.md [hunk 3]
```diff
@@ -122,3 +122,76 @@
...
+- Team trends around practices (e.g. trends in testing and quality, security and privacy practices - e.g. Apple positioning)
+- Discerning about tradeoffs with vendors and effect of their business model on our scaling (how to think about Algolia vs Elastic Search models)
+
+This experience and connectedness to technology trends enables a team member to contribute more effectively
+in planning and in system design activities by identifying future needs and the technology and vendor trends 
+that are likely to be tailwinds or headwinds to our technical delivery. 
+
+#### Examples of Authority
+##### Real Estate
+
```
Comment (mruthven) [inline competencies/roles/_TechWideGeneral.md:195]: Would we consider basic understanding of CM, and the various components that go into CM1, 2, 3, 4, to be demonstration of proficiency or authority? I might lean towards authority for a technical contributor, just checking here.

Comment (rsiu) [reply competencies/roles/_TechWideGeneral.md:195]: I think "Understands unit economics of the business" implies we'd expect basic understanding of the CM indicators at the Proficient level

**Review by mruthven — APPROVED** [review]

**Pull request merged**